src/index.tsx

```diff
@@ -1,11 +1,11 @@
-import React from "react";
-import ReactDOM from "react-dom/client";
-import "./index.css";
-import App from "./App";
-import reportWebVitals from "./reportWebVitals";
-import "../node_modules/@fortawesome/fontawesome-free/css/all.min.css";
+import React from 'react';
+import ReactDOM from 'react-dom/client';
 
-const root = ReactDOM.createRoot(document.getElementById("root"));
+import App from './App';
+import reportWebVitals from './reportWebVitals';
+import '@fortawesome/fontawesome-free/css/all.min.css';
+
+const root = ReactDOM.createRoot(document.getElementById('root') as HTMLElement);
```

Comment:
it's better to throw an error if `document.getElementById('root')` is undefined instead of just casting its type and assert it's not null